gcp-deployment-manager/templates/docksal_template.jinja

```diff
@@ -0,0 +1,56 @@
+{% set staticIpName = "static-ip-%s" % env["deployment"] %}
+{% set project = env["project"] %}
+{% set region = properties["region"] %}
+{% set templateId = properties["templateId"] %}
+{% set deployment = env["deployment"] %}
+{% set preemptive = properties["preemptive"] %}
+
+resources:
+- name: {{ staticIpName }}
+  properties:
+    addressType: EXTERNAL
+    description: |
+      The static IP address used for VMs in the deployment
+      {{ deployment }}.
+    region: {{ region }}
+    resourceType: addresses
+  type: gcp-types/compute-v1:addresses
+- name: {{ templateId }}
+  properties:
+    name: {{ templateId }}
+    properties:
+      machineType: {{ properties["machineType"] }}
+      disks:
+      - deviceName: boot
+        boot: true
+        autoDelete: true
+        initializeParams:
+          sourceImage: projects/ubuntu-os-cloud/global/images/ubuntu-1804-bionic-v20200610
```

Comment:
Switch to https://www.googleapis.com/compute/v1/projects/ubuntu-os-cloud/global/images/ubuntu-minimal-2004-focal-v20200610. This will require using `DOCKSAL_VERSION=develop` for now, so it makes sense to have `DOCKSAL_VERSION` available as a config value.

Reply:
Updated.

Reply:
Actually updated this time. You can specify DOCKSAL_VERSION in project.env and it's classified as overridable.